kit/ForKit.cpp

```diff
@@ -327,23 +362,25 @@ static void cleanupChildren()
                                           << childJails.size() << " left: " << oss.str());
     }
 
-    if (segFaultCount)
-    {
+    if (segFaultCount || killedCount || oomKilledCount)
         if (WSHandler)
         {
-            std::stringstream stream;
-            stream << "segfaultcount " << segFaultCount << '\n';
-            int ret = WSHandler->sendMessage(stream.str());
-            if (ret == -1)
             {
-                LOG_WRN("Could not send 'segfaultcount' message through websocket");
-            }
-            else
-            {
-                LOG_WRN("Successfully sent 'segfaultcount' message " << stream.str());
+                std::stringstream stream;
+                stream << "faultcount " << segFaultCount << ' ' << killedCount << ' '
```

Comment:
much better =) though we could use keys for that faultcount foo=3 baa=5 oom=1 - like most of the other commands we use, and then have it be more descriptive and not need the new StringToken method (?) =)

Reply:
Done